libs/crosstarget-utils/src/common.rs

```diff
@@ -21,3 +21,24 @@ impl Display for TimeoutError {
 }
 
 impl std::error::Error for TimeoutError {}
+
+#[derive(Debug)]
+pub struct RegExpError {
+    pub message: String,
+}
+
+#[derive(PartialEq)]
+pub enum RegExpFlags {
+    IgnoreCase,
+    Multiline,
+}
+
+impl From<RegExpFlags> for String {
```

Comment:
I'd rather this was a `Display` impl for `RegExpFlags` instead.

Or, even better, a custom `as_str()` method that returns a `&'static str` so we don't allocate here.

Reply:
with a bitmask it will be a `Display` impl after all since we'll be converting the whole mask at once